fundamentals/code-quality/.vitepress/theme/components/OneNavigation.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div class="one-navigation">
+    <div class="nav-top">
+      <div
+        v-for="item in ONE_NAVIGATION_ITEMS"
+        :key="item.path"
+        class="nav-item"
+        :class="{ active: isActive(item.path) }"
+        :data-tooltip="(isKorean ? item.tooltip.ko : item.tooltip.en)"
+      >
+        <a :href="item.href">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 24 24"
+            fill="currentColor"
+            v-html="item.icon"
+          ></svg>
+        </a>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { useRoute } from "vitepress";
+import { useLocale } from "../composables/useLocale";
+import { ONE_NAVIGATION_ITEMS } from "../config/OneNavigationItems";
+
+const route = useRoute();
+const { isKorean } = useLocale();
+
+function isActive(path: string): boolean {
+  return route.path.startsWith(path);
+}
+</script>
+
+<style scoped>
+.one-navigation {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 50px;
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  border-right: 1px solid var(--vp-c-divider);
+  z-index: 40;
+  background-color: var(--vp-c-bg-alt);
+  padding-top: var(--vp-nav-height);
+}
+
+.nav-top {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 1.5rem;
+  width: 100%;
+}
+
+.nav-bottom {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-bottom: 1.5rem;
+  width: 100%;
+  margin-bottom: 1rem;
+}
+
+.nav-item {
+  width: 100%;
+  height: 36px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 0.5rem;
+  position: relative;
+}
+
+.nav-item a {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+  color: var(--vp-c-text-2);
+  transition: color 0.2s;
+}
+
+.nav-item svg {
+  width: 20px;
+  height: 20px;
+}
+
+.nav-item:hover a {
+  color: var(--vp-c-text-1);
+}
+
+.nav-item.active a {
+  color: #474747;
+}
```

Comment:
how about changing the active state color on the button? 
current color doesn’t seem to stand out well in dark mode :eyes:

```suggestion
.nav-item.active a {
  color: var(--vp-c-brand);
}
```

AS-IS
<img width="1511" alt="image" src="https://github.com/user-attachments/assets/1f2f0350-1b67-4b6e-b28d-60f859b3801c" />

TO-BE
<img width="1431" alt="image" src="https://github.com/user-attachments/assets/75cc0709-a06b-44e8-b0bb-6f61fe4817c2" />

Reply:
👍👍👍

Reply:
applied in [faet: update as adative color](https://github.com/toss/frontend-fundamentals/pull/192/commits/74812666fc6ee3bf7f9f9385937e906260e79276)